x-pack/plugin/sql/sql-proto/src/main/java/org/elasticsearch/xpack/sql/proto/Mode.java

```diff
@@ -17,7 +17,7 @@
     ODBC;
 
     public static Mode fromString(String mode) {
-        if (mode == null) {
+        if (mode == null || mode.isEmpty()) {
```

Comment:
Potentially `Strings.isEmpty`

Reply:
There is no utility class visible to this class that has an `isEmpty()`  method useful here. Or I missed it... which is very likely.